Dockerfile

```diff
@@ -19,4 +19,6 @@ RUN mkdir -p $APP_HOME
 ADD . $APP_HOME
 WORKDIR $APP_HOME
 
-RUN yarn && yarn build
```

Comment:
Maybe add a comment here to explain what it's doing, similar to the other commands above.